src/cr/cube/cubepart.py

```diff
@@ -1090,18 +1090,7 @@ def rows_scale_median(self):
         if not self._columns_have_numeric_value:
             return None
```

Comment:
The `._columns_have_numeric_value` property should probably move down, at least as far as `Assembler` but I think probably down to the `ScaledCounts` class because that's who would naturally need to know what the numeric values were and be able to easily test whether there were any.